### PR TITLE
Hammoud/add accelerometer

### PR DESCRIPTION
**Pull request by hammoudbilal**
This pull request adds the following to `SimHead`

1. accelerometer and gyroscope readings from bullet with noise and stores them in `imu_gyroscope` and `imu_accelerometer` 
2. contact forces and contact status reading from pybullet according to the pinocchio convention and stores them in   
   - `SimHead._sensor__force_plate_force`
   - `SimHead._sensor__force_plate_status`
   This can be used in simulation the same way the vicon is simulated

[//]: # "Thanks for your contribution.  To make life of the reviewers easier,"
[//]: # "please give this pull request a meaningful title and provide the"
[//]: # "requested information below."

## Description

[//]: # "Description of what you did.  If it is more complex, consider to add a"
[//]: # "'Summary' section."

## How I Tested

[//]: # "Explain how you tested your changes"

## Do not merge before

[//]: # "Link other open pull request here that need to be merged first."
[//]: # "Remove this section if it does not apply."

- [ ] link to dependent pull request

## I fulfilled the following requirements

[//]: # "Please make sure you followed these steps before requesting a review."
[//]: # "Check the boxes in the list below, when done."

- [ ] All new code is formatted according to our style guide (for C++ run clang-format, for Python, run flake8 and fix all warnings).
- [ ] All new functions/classes are documented and existing documentation is updated according to changes.
- [ ] No commented code from testing/debugging is kept (unless there is a good reason to keep it).

### CODE REVIEW

File: src/dynamic_graph_head/sim_head.py [hunk 1]
```diff
@@ -6,6 +6,7 @@
 All rights reserved.
 """
 
+from os import write
```
Comment (jviereck) [inline src/dynamic_graph_head/sim_head.py]: Why do you import the `write` function?

File: src/dynamic_graph_head/sim_head.py [hunk 2]
```diff
@@ -105,12 +111,19 @@ def read(self):
...
+            self._sensor_imu_accelerometer[:] = history['imu_accelerometer'][read_idx]
             self._sensor__vicon_base_position[:] = q[:7]
             self._sensor__vicon_base_velocity[:] = dq[:6]
+            # only read forces for free floating for now 
+            contact_status, contact_forces = self._robot.get_force()
+            for i, cnt_id in enumerate(self._robot.pinocchio_endeff_ids):
+                self._sensor__force_plate_force[i,:] = contact_forces[i][:]
+                self._sensor__force_plate_status[i] = contact_status[i] 
+
+
```
Comment (jviereck) [inline src/dynamic_graph_head/sim_head.py]: Remove extra newline here (only single newline please)

File: src/dynamic_graph_head/vicon.py [hunk 3]
```diff
@@ -1,4 +1,4 @@
-"""__init__
+"""
```
Comment (jviereck) [inline src/dynamic_graph_head/vicon.py:1]: Good catch :)

File: src/dynamic_graph_head/force_plate.py [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+"""
+#NOTE: This is only a simulation class for now, we don't have an actual force plate 
+
+from matplotlib.pyplot import hist
+import numpy as np
+import pinocchio as pin
+
+
+class SimForcePlate:
+    def __init__(self, objects, delay_dt=0, noise_data_std={}):
```
Comment (jviereck) [inline src/dynamic_graph_head/force_plate.py]: As `noise_data_std` is not implemented yet, maybe it's best to not provide it as an argument? This way, if someone provides the data they don't get confused why it's not getting applied.

File: src/dynamic_graph_head/sim_head.py [hunk 5]
```diff
@@ -105,12 +111,19 @@ def read(self):
...
             self._sensor_imu_gyroscope[:] = history['imu_gyroscope'][read_idx]
-
+            self._sensor_imu_accelerometer[:] = history['imu_accelerometer'][read_idx]
             self._sensor__vicon_base_position[:] = q[:7]
             self._sensor__vicon_base_velocity[:] = dq[:6]
+            # only read forces for free floating for now 
+            contact_status, contact_forces = self._robot.get_force()
+            for i, cnt_id in enumerate(self._robot.pinocchio_endeff_ids):
+                self._sensor__force_plate_force[i,:] = contact_forces[i][:]
+                self._sensor__force_plate_status[i] = contact_status[i]
```
Comment (jviereck) [inline src/dynamic_graph_head/sim_head.py:123]: I am not sure putting more and more fields on the sim-head that are then read/processed by other objects is a good idea. But maybe for now we can keep it.

File: examples/solo12_centroidal.ipynb [hunk 6]
```diff
@@ -2,9 +2,17 @@
  "cells": [
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 1,
```
Comment (jviereck) [inline examples/solo12_centroidal.ipynb:5]: Are there any real changes to this file? If not, can you avoid updating the file in this commit?

**Review by jviereck — CHANGES_REQUESTED** [review]
This looks good overall - thanks for your contributions!

Can you add a .py demo file showing how to use the SimForcePlate as well? Something simple like applying PD at the joints using the HoldPDController and reading the force measurement.

File: src/dynamic_graph_head/sim_head.py (hunk 2 above)
Comment (hammoudbilal) [reply src/dynamic_graph_head/sim_head.py]: alright, removed

File: src/dynamic_graph_head/sim_head.py (hunk 1 above)
Comment (hammoudbilal) [reply src/dynamic_graph_head/sim_head.py]: i don't even remember adding this, removed it

File: src/dynamic_graph_head/force_plate.py (hunk 4 above)
Comment (hammoudbilal) [reply src/dynamic_graph_head/force_plate.py]: you're right! and i fixed that

File: src/dynamic_graph_head/sim_head.py (hunk 5 above)
Comment (hammoudbilal) [reply src/dynamic_graph_head/sim_head.py:123]: I tried to follow the same way you added the vicon to avoid different logic

File: examples/.jovianrc [hunk 7]
```diff
@@ -0,0 +1,3 @@
+{
+  "notebooks": {}
+}
```
Comment (jviereck) [inline examples/.jovianrc]: Is this file needed? Can it be removed?

**Review by jviereck — CHANGES_REQUESTED** [review]
